tests/test_contrib.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2016-2020 CERN.
+#
+# Marshmallow-Utils is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Test the marshmallow contrib."""
+
+import pytest
+from marshmallow import Schema, ValidationError, fields
+
+from marshmallow_utils.fields import contrib
+
+
+def serialize_foo(obj, context):
+    return {'serialize_args': {'obj': 1, 'context': 2}}
+
+
+def deserialize_foo(value, context, data):
+    return {'deserialize_args': {
+        'value': 3, 'context': 4, 'data': 5}}
+
+
+class MySchema(Schema):
+    f = contrib.Function(serialize_foo, deserialize_foo)
+
+
+assert MySchema().dump({'f': 12123}) == {'f': {'serialize_args':
+                                         {'context': 2, 'obj': 1}}}
+
+assert MySchema().load({'f': 12123}) == {'f': {'deserialize_args': {
+                                        'value': 3, 'context': 4,
+                                         'data': 5}}}
+
+
+class BarSchema(Schema):
+    bar = contrib.Method('serialize_bar', 'deserialize_bar')
+
+    def serialize_bar(self, obj):
+        return {'serialize_args': {'obj': 1}}
+
+    def deserialize_bar(self, value, data):
+        return {'deserialize_args': {'value': 23, 'data': 24}}
+
+
+assert BarSchema().dump({'bar': 12123}) == {'bar': {'serialize_args':
+                                            {'obj': 1}}}
+
+
+assert BarSchema().load({'bar': 12123}) == {'bar': {'deserialize_args':
+                                            {'value': 23, 'data': 24}}}
```

Comment:
It turns out that the documentation for Function and Method (https://github.com/inveniosoftware/marshmallow-utils/blob/master/marshmallow_utils/fields/contrib.py#L36-L65, https://github.com/inveniosoftware/marshmallow-utils/blob/master/marshmallow_utils/fields/contrib.py#L93-L118) has what is essentially a test case to be put into an actual test (add `assert` and add a case where the context is full)

Interestingly, you will notice that the documented usage of Function is buggy. So you can fix it in passing:

`FooSchema().dump({'foo': 42}).data` -> `FooSchema().dump({'foo': 42})`.

Reply:
I have changed the documentation!